### PR TITLE
added images

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-afbeeb9313/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-8a53dcb2f1/how-to/)

**Comment by stephamon** [comment]
One more @cyrichardson though it's very similar to 2370!

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a4bc23efa4/how-to/)

**Comment by cyrichardson** [comment]
@stephamon Looking now.

File: content/exchange/safelist-senders-in-owa.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+For more information about prerequisite terminology, see [Cloud Office support terminology](/how-to/cloud-office-support-terminology/).
+
+### Safelist Senders
+
+Safelisting does not prevent a sender from being flagged as spam. Safelisting ensures that a sender's message is delivered to your inbox, even if the sender is flagged as spam, so that messages from a trusted source aren't overlooked in the junk folder. 
+
+For administrator instructions on safelisting senders through the control panel, see [Blacklists in Microsoft Exchange](/how-to/spam-preferences-safe-lists-and-black-list-in-microsoft-exchange/#managing-safe-list).
+
+1. Log in to your Exchange mailbox at [apps.rackspace.com](apps.rackspace.com)
```
Comment (cyrichardson) [inline content/exchange/safelist-senders-in-owa.md]: Add a period at the end.

**Comment by cyrichardson** [comment]
@stephamon One minor thing.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-02bf590a4a/how-to/). :bow:

**Pull request merged**